tutor/commands/upgrade/compose.py

```diff
@@ -168,7 +179,17 @@ def upgrade_mongodb(
         )
         return
 
+    # Just need the main release version for certain checks
+    mongo_version = int(to_docker_version[0])
+    admin_command: t.Dict[str, t.Any] = {
```

Comment:
This is actually a `dict[str, str]` and there is no need to `import typing as t`. Same below in the k8s module.

Reply:
When "confirm": 1 is added, the value is int and then quality checks fail.

Reply:
Then we need the typing import, but not `Any`, because we can use `t.Union[int, str]` as a value type.